src/Microsoft.Data/StringColumn.cs

```diff
@@ -9,7 +9,7 @@
 namespace Microsoft.Data
 {
     /// <summary>
-    /// A column to hold strings
+    /// A mutable column to hold strings. Is NOT Arrow compatible
```

Comment:
Something like `Is NOT Arrow compatible` is more appropriate for a `<remarks>` section, not a summary.